### ISSUE TITLE
FactoryInterface not added in "use" when configuration option "'is_factory'    => true," is set

### ISSUE THREAD

**Issue by stevleibelt** [bazzline/php_component_locator_generator#2]
```
    'instances' => array(
        array(
           //...
            'is_factory'    => true,
```

Configuration above does not adds the "use Net\Bazzline\Component\Locator\FactoryInterface" to the generated locator

**Comment by stevleibelt** [comment]
Fixed with commit [a7dacd614cecd1a7c890ccd44e309d5945f8788a](https://github.com/bazzline/php_component_locator_generator/commit/a7dacd614cecd1a7c890ccd44e309d5945f8788a).

**Comment by stevleibelt** [comment]
Fixed in release [2.0.0](https://github.com/bazzline/php_component_locator_generator/tree/2.0.0)
